graphql_compiler/compiler/subclass.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017-present Kensho Technologies, LLC.
+from graphql.type.definition import GraphQLObjectType, GraphQLUnionType
+import six
+
+
+def _transitive_closure(graph):
+    """Compute the transitive closure of a graph represented as dict inplace."""
+    # Floyd-Warshall O(N^3)
+    for k, k_out in six.iteritems(graph):
+        for i, i_out in six.iteritems(graph):
+            for j, j_out in six.iteritems(graph):
+                if k in i_out and j in k_out:
+                    graph.setdefault(i, set()).add(j)
+    return graph
+
+
+def compute_subclass_sets(schema, type_equivalence_hints):
+    """Return a dict mapping class names to the set of its subclass names."""
+    subclass_set = dict()
+
+    for classname, cls in six.iteritems(schema.get_type_map()):
```

Comment:
Don't use `cls`, its canonical use is for Python class objects and you can see that the syntax highlighting picks it up.